index.html

```diff
@@ -1,18 +1,147 @@
-<script>
-let name = document.location.pathname.slice(1);
-
-if (!name) {
-  console.log("no place name specfied");
-} else {
-  console.log("fetching place named '" + name + "'");
-
-  fetch(name + '.json')
-    .then(function (response) {
-      return response.text();
-    })
-    .then(function (myText) {
-      console.log(myText);
-    });
-}
-
-</script>
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Yelpington</title>
+    <link rel= 'stylesheet' href="style.css"/>
+		<link
+			rel="stylesheet"
+			href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css"
+		/>
+    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+	</head>
+
+	<body>
+    <div id = 'title'>Yelpington</div>
+		<div id="map"></div>
+		<header id="restaraunt">
+			<ul>
+				Restaraunt List
+				<li>
+					<a href="http://localhost:5500/restaurant.html#ahli-babas-kabob"
+						>Ahli Babas Kabob Shop</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:5500/restaurant.html#american-flatbread"
+						>American Flatbread</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:5500/restaurant.html#august-first"
+						>August First</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:5500/restaurant.html#city-market"
+						>City Market</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:5500/restaurant.html#el-cortijo"
+						>El Cortijo</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:5500/restaurant.html#farmhouse-grill"
+						>Farmhouse Grill</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:5500/restaurant.html#hen-of-the-wood"
+						>Hen of The Wood</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:5500/restaurant.html#honey-road"
+						>Honey Road</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:5500/restaurant.html#kountry-kart-deli"
+						>Kountry Kart Deli</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:5500/restaurant.html#mr-mikes">Mr. Mikes</a>
+				</li>
+				<li>
+					<a href="http://localhost:5500/restaurant.html#pascolo-ristorante"
+						>Pascolo Ristorante</a
+					>
+				</li>
+			</ul>
+		</header>
+
+		<script>
```

Comment:
From an organizational standpoint I would like to see the javascript split into its own JS file.